docs/profiling.md

```diff
@@ -13,8 +13,8 @@ GPU and TPU. The end result looks something like this:
 ### Installation
 
```

Comment:
Hi @bchetioui 👋  Thank you so much for updating the JAX profiler docs. We're stuck at https://github.com/google/jax/issues/5069 as you probably are aware. And workarounds, such as https://github.com/google/jax/issues/4427, have helped some folks here.

May I suggest a small thing here from the UX perspective for users who aren't as experienced with TB:

```suggestion

Install specific nightly versions of TensorBoard, TensorFlow, as well as the What-If Tool TensorBoard:
```